frontend/src/component/pages/Home.tsx

```diff
@@ -82,6 +89,16 @@ export class Home extends Component<Props, State> {
     });
     this.handleStateChange();
     this.autoFillLongLink();
+    const changeLog = await this.props.updatesService.getChangeLog();
+    this.setState({
+      changeLog
+    });
+    const lastSeenChangeLog = await this.props.updatesService.getLastSeenChangeLog();
+    if (changeLog && lastSeenChangeLog < changeLog[0].publishedAt) {
+      this.setState({
```

Comment:
If change log last seen epoch time is before new published article epoch time then prop drill newUpdateReleased (after setting it to true) down to ChangeLogModal component to open the modal.

Reply:
newUpdateReleased => newUpdateReleased => showChangeLogModal => shouldShowModal

Reply:
Home => Footer => ChangeLogModal